data.sql

```diff
@@ -21,4 +21,15 @@ VALUES (9, 'Boarmon', '2005-06-07',7, true,20.4);
 INSERT INTO animals (ID,NAME,date_of_birth,escape_attempts,neutered,weight_kg) 
 VALUES (10, 'Blossom', '1998-10-13',3, true,17);
 INSERT INTO animals (ID,NAME,date_of_birth,escape_attempts,neutered,weight_kg) 
-VALUES (11, 'Ditto', '2022-05-14',4, true,22)
\ No newline at end of file
+VALUES (11, 'Ditto', '2022-05-14',4, true,22);
+
+-- insertion to owners table
+INSERT INTO owners(full_name,age) VALUES('Sam Smith',34);
+INSERT INTO owners(full_name,age) VALUES('Jennifer Owl',19);
+INSERT INTO owners(full_name,age) VALUES('Bob',45);
+INSERT INTO owners(full_name,age) VALUES('Melody Pond',77);
+VALUES VALUES owners(full_name,age) VALUES('Dean Whinchester',14);
+VALUES INTO owners(full_name,age) VALUES('Jodie Whittaker',38);
```

Comment:
- [ ] In line `31` and line `32`, kindly correct the insert statements as the current syntax is incorrect.